.github/workflows/i18n.yml

```diff
@@ -0,0 +1,33 @@
+name: I18n
+
+on:
+  schedule:
+    - cron: '0 6,18 * * *'
```

Comment:
This does not appear to have run over the weekend, so something must not be configured right here...

Reply:
Ah, it looks like scheduled actions will only run on the [most recent commit on the default branch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events).